File: app/src/main/java/io/github/a13e300/myinjector/bridge/LoadPackageParam.java
package io.github.a13e300.myinjector.bridge;

import android.content.pm.ApplicationInfo;

public class LoadPackageParam {
    public final String packageName;
    public final String processName;
    public final ClassLoader classLoader;
    public final ApplicationInfo appInfo;
    public final boolean isFirstApplication;

    public LoadPackageParam(String packageName, String processName, ClassLoader classLoader, ApplicationInfo info, boolean isFirst) {
        this.packageName = packageName;
        this.processName = processName;
        this.classLoader = classLoader;
        appInfo = info;
        isFirstApplication = isFirst;
    }
}

File: app/src/main/java/io/github/a13e300/myinjector/bridge/LoadPackageParam.kt
package io.github.a13e300.myinjector.bridge

import android.content.pm.ApplicationInfo

class LoadPackageParam(param: de.robv.android.xposed.callbacks.XC_LoadPackage.LoadPackageParam) {
    val packageName: String = param.packageName
    val processName: String = param.processName
    val classLoader: ClassLoader = param.classLoader
    val appInfo: ApplicationInfo = param.appInfo
    val isFirstApplication: Boolean = param.isFirstApplication
}
